plugins/skipmap/index.js

```diff
@@ -0,0 +1,158 @@
+import { CHAT_MESSAGE, NEW_GAME } from 'squad-server/events';
+
+export default {
+  name: 'skipmap',
+  description:
+    'The `skipmap` plugin will allow players to vote via `+/-` if they wish to skip the map',
```

Comment:
Replace \` with `<code>...</code>`. The recent documentation changes stoped \` from working.

I'd state the "current" map. Skipping the next map may be something we want to add in the future.